### ISSUE TITLE
Aggiornamento API NUTS

### ISSUE THREAD

**Issue by SirTakobi** [OCA/l10n-italy#3366]
## Module
l10n_it_location_nuts

## Describe the bug
L'errore è 
> Got an error 404 when trying to download the file http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_CLS_DLD&StrNom=NUTS_2013&StrFormat=XML&StrLanguageCode=EN&StrLayoutCode=HIERARCHIC..

## To Reproduce
**Affected versions**:
- [x] `12.0`: sì (risolta per termine supporto)
- [ ] `14.0`: sì
- [x] `16.0`: https://github.com/OCA/l10n-italy/pull/4458

1. Eseguire i test automatici

**Expected behavior**
I test hanno successo.

**Additional context**
Normalmente è un errore temporaneo, ma sembra che questa volta ci sia in ballo qualche cambiamento nelle API, infatti la pagina del link riporta:

> The [bulk download](https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/BulkDownloadListing) will no longer be available as of October 2023.
Users are invited to start using the new features that are available as replacement:
>
>- Manual downloads using the data browser ['Download operations'](https://ec.europa.eu/eurostat/databrowser/bulk?lang=en) section
>- Automatic downloads using the new API. For details, please consult the ['developer's corner'](https://wikis.ec.europa.eu/display/EUROSTATHELP/Developer%27s+corner) which is part of the data browser online help section. The chapter ['Transition - from Eurostat bulk download to API'](https://wikis.ec.europa.eu/display/EUROSTATHELP/Transition+-+from+Eurostat+Bulk+Download+to+API) contains important information if you need to perform a migration.
>
>For any additional questions, please contact [user support](https://ec.europa.eu/eurostat/web/main/contact-us/user-support).

Non so come mai stia già dando errore, ma è possibile che ci sarà da cambiare qualcosa qui e/o in https://github.com/OCA/partner-contact/tree/36d0c7bf630d63137f086f969c0201f25718ae9e/base_location_nuts

**Comment by francesco-ooops** [comment]
@odooNextev penso che sia da risolvere in partner-contact: https://github.com/OCA/partner-contact/actions/runs/5197286531/jobs/9371927563?pr=1533#step:8:462

**Comment by matteoopenf** [comment]
Qualcuno sta lavorando al problema?

**Comment by francesco-ooops** [comment]
@matteoopenf no, sarebbe da aprire una issue/PR in partner-contact dove stanno fallendo i test, eg: https://github.com/OCA/partner-contact/actions/runs/5517446432/jobs/10060074798?pr=1553#step:8:453

**Comment by matteoopenf** [comment]
> @matteoopenf no, sarebbe da aprire una issue/PR in partner-contact dove stanno fallendo i test, eg: https://github.com/OCA/partner-contact/actions/runs/5517446432/jobs/10060074798?pr=1553#step:8:453

concordo

**Comment by francesco-ooops** [comment]
Il merge di https://github.com/OCA/partner-contact/pull/1826 è necessario per riattivare/riadattare i test

**Comment by matteoopenf** [comment]
hanno mergiato quella per 14 anche oltre che per 16

**Comment by francesco-ooops** [comment]
@matteoopenf riesci a fare una PR per riattivare i test e vedere cosa succede?
